lib/src/components/AbstractDialog/__tests__/AbstractDialog.test.tsx

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { render, fireEvent, waitFor, createEvent } from '@testing-library/react';
+import { act } from 'react-dom/test-utils';
+import { screen } from '@testing-library/dom';
+import userEvent from '@testing-library/user-event';
+
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
+
+import AbstractDialog, { Props } from '../AbstractDialog';
+import NxButton from '../../NxButton/NxButton';
+import NxTooltip from '../../NxTooltip/NxTooltip';
+import useToggle from '../../../util/useToggle';
+
+describe('AbstractDialog', function() {
+  const minimalProps: Props = {
+    children: 'A message inside the dialog element.',
+    onCancel: jest.fn()
```

Comment:
Just use `() => {}` here, it isn't really safe to use spies that aren't reset per-test (e.g. in `beforeEach` or in an individual test).  You can redefine `onCancel` as a spy in the tests that need it to be one

Reply:
> Just use `() => {}` here, it isn't really safe to use spies that aren't reset per-test (e.g. in `beforeEach` or in an individual test). You can redefine `onCancel` as a spy in the tests that need it to be one

Fixed: b6275efc97b38b00baefc0749659a4cac32444fc